db/models/address.js

```diff
@@ -0,0 +1,48 @@
+const { STRING, INTEGER } = require('sequelize');
+
+module.exports = db => db.define('addresses', {
+  street: {
+    type: STRING,
+    allowNull: false,
+    validate: {
+      isAlphanumeric: true
```

Comment:
I looked online for a regexp for the address validation and viewed a few stackoverflows. I found plenty of valid regexps, but they don't catch all addresses and have plenty of false-positives. One comment suggested not having address validation, so I think that that's a good idea for now.